src/main/webapp/app/localvc/repository-view/repository-view.component.ts

```diff
@@ -86,6 +87,9 @@ export class RepositoryViewComponent implements OnInit, OnDestroy {
             const exerciseId = Number(params['exerciseId']);
             const participationId = Number(params['participationId']);
             this.repositoryType = participationId ? 'USER' : params['repositoryType'];
+            this.routeVcsAccessLog = this.router.url + '/vcs-access-log';
+            this.vcsAccessLogEnabled = this.router.url.includes('course-management') && params['repositoryType'] != 'TESTS';
```

Comment:
```suggestion
            this.vcsAccessLogEnabled = this.router.url.includes('course-management') && params['repositoryType'] !== 'TESTS';
```

Also, there are different permissions for these VCS access logs:

For paths with a repositoryType:
            authorities: [Authority.ADMIN, Authority.INSTRUCTOR, Authority.EDITOR],

For paths without a repositoryType:
            authorities: [Authority.ADMIN, Authority.INSTRUCTOR],

Please make sure the button is only displayed if these prerequisites are met